README.md

```diff
@@ -1,8 +1,5 @@
 # Getting Started with the .NET MAUI SignaturePad 
-A quick-start project that helps you integrate the Signature Pad control with the .NET MAUI application by adding the Syncfusion.Maui.SignaturePad package from NuGet.
-
-Refer to the following documentation for the Syncfusion .NET MAUI SignaturePad Control: 
-https://help.syncfusion.com/maui/signaturepad/getting-started
+The [.NET MAUI SignaturePad](https://www.syncfusion.com/maui-controls/maui-signaturepad?utm_source=github&utm_medium=listing&utm_campaign=maui-signaturepad-github-samples) is an interactive UI control that allows users to capture smooth and realistic signatures. Also, users can save their signatures as images and sync them across devices and documents that need signatures. A quick-start project that helps you integrate the Signature Pad control with the .NET MAUI application by adding the Syncfusion.Maui.SignaturePad package from NuGet.
 
 ## Project prerequisites
 Make sure that you have the compatible versions of [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/ ) with the Dot NET MAUI workload and [.NET Core SDK 7.0](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) or later version in your machine before starting to work on this project.

```

Comment:
Remove extra space here
![image](https://github.com/SyncfusionExamples/getting-started-with-the-dotnet-maui-signature-pad/assets/95272441/14e56e89-10de-4dd8-a1d8-fb732d2e2bf1)